tests/io/v3/vm/test_agent_groups.py

```diff
@@ -0,0 +1,186 @@
+'''
+test plugins
+'''
+import re
+
+import pytest
+import responses
+
+BASE_URL = 'https://cloud.tenable.com/api/v3/agent-groups'
+
+
+@responses.activate
+def test_add_agent_with_single_agent_id(api):
+    '''Test case for add_agent method with sigle agent id'''
+    group_id: int = 470009
+    agent_id: int = 5
+    responses.add(
+        responses.PUT,
+        re.compile(f'{BASE_URL}/{group_id}/agents/{agent_id}')
+    )
+    res = api.v3.vm.agent_groups.add_agent(group_id, agent_id)
+    assert res is None
+
+
+@responses.activate
+def test_add_agent_with_multiple_agent_id(api):
+    '''Test case for add_agent method with multiple agent id'''
+    group_id: int = 470009
+    payload = {'items': [i for i in (1, 2, 3, 4)]}
+    test_response = {
+        'task_id': '07a665f4-6e09-444b-b9ce-ed5ccf5c193b',
+        'container_uuid': 'cfdabb09-6aef-481d-b28f-aecb1c38f297',
```

Comment:
This should be container_id vs container_uuid currently.

Reply:
Done.